### PR TITLE
Booking mo feature

### PR DESCRIPTION
**Pull request by arman7rm**
Finished search result component

### CODE REVIEW

File: angular/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   },
   "devDependencies": {
     "@angular-builders/custom-webpack": "^9.1.0",
-    "@angular-devkit/build-angular": "~0.901.0",
+    "@angular-devkit/build-angular": "^0.1000.5",
```
Comment (dakatk) [inline angular/package.json]: Keep this at Angular version 9

File: angular/src/app/modules/booking/booking-routing.module.ts [hunk 2]
```diff
@@ -4,7 +4,9 @@ import { RouterModule, Routes } from '@angular/router';
 import { BookingComponent } from './booking/booking.component';
 import { SearchResultsComponent } from './search-results/search-results.component';
 
-const routes: Routes = [{ component: BookingComponent, path: '' }];
+const routes: Routes = [
+  { component: BookingComponent, path: '' }];
+
```
Comment (dakatk) [inline angular/src/app/modules/booking/booking-routing.module.ts]: Are you using Prettier? Is this a result of that or the default VSCode formatter?

File: angular/src/app/modules/booking/booking/booking.component.ts [hunk 3]
```diff
@@ -11,6 +11,8 @@ import { LodgingService } from 'src/app/services/lodging/lodging.service';
 export class BookingComponent implements OnInit {
   lodgings$: Observable<Lodging[]>;
```
Comment (dakatk) [inline angular/src/app/modules/booking/booking/booking.component.ts]: No newline

File: angular/src/app/modules/booking/booking/booking.component.ts [hunk 4]
```diff
@@ -11,6 +11,8 @@ import { LodgingService } from 'src/app/services/lodging/lodging.service';
...
 
+  isSearched : boolean = false;
```
Comment (gomez714) [inline angular/src/app/modules/booking/booking/booking.component.ts]: The search bar will be taking in an object, so this will probably change.

**Review by dakatk — CHANGES_REQUESTED** [review]
Some formatting fixes

File: angular/tslint.json [hunk 5]
```diff
@@ -1,4 +1,5 @@
 {
+  "strictTemplates": false,
```
Comment (gomez714) [inline angular/tslint.json:2]: Does this need to be here?

**Review by jgamble482 — APPROVED** [review]
Looks good bro. Other homies already covered everything I'd say (strictTemplate set to true in tslint).

**Pull request merged**